src/components/editor/editor.css

```diff
@@ -0,0 +1,13 @@
+.text-editor-wrapper{
+  width: 100vh;
+  height: 30vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.text-editor{
+    width: 58.2rem;

```

Comment:
Instead of using rems here, try to use percentage. Giving width in decimals is not a nice idea.